cabal.project.dist

```diff
@@ -1,3 +1,13 @@
+-- See https://github.com/Happstack/boomerang/issues/12
+allow-newer: boomerang:template-haskell
+-- See https://github.com/GaloisInc/bv-sized-float/issues/8
+allow-newer: bv-sized-float:containers
+-- https://github.com/GaloisInc/grift/issues/13
+allow-newer: grift:containers
+allow-newer: grift:filepath
```

Comment:
I don't mind bumping the version bounds for `boomerang` and `haggle` for now (as those are outside of the Galois Cinematic Universe), but I wonder if we shouldn't bump the version bumps downstream in `bv-sized-float` and `grift` first before landing this. We control the latter two packages, so it ought to be much more straightforward to handle them.